website/docs/commands/init.html.markdown

```diff
@@ -68,6 +68,9 @@ The command-line flags are all optional. The list of available flags are:
   ignore previously-downloaded objects and install the latest version allowed
   within configured constraints.
 
+* `-verify-plugins=true` Verify the authenticity and integrity of automatically
+                         downloaded plugins.
```

Comment:
The line wrapping here is formatted differently than the ones above. I guess it doesn't really matter since this is markdown, but it'd be nice to stay consistent.